include/cucumber-cpp/internal/step/StepManager.hpp

```diff
@@ -62,6 +62,11 @@ class InvokeArgs {
 
     template<class T> T getInvokeArg(size_type i) const;
     const Table & getTableArg() const;
+
+    size_type argsSize() const {
+      return args.size();
+    }
+
```

Comment:
Apart from the Boost improvements that should have come from another PR, I think this is the only change that I didn't apply in #107. Once that is merged, it's probably easier to apply this single change on top of master instead of rebasing (that took me quite some time).